index.js

```diff
@@ -47,7 +63,7 @@
     if (src && src !== '') {
```

Comment:
Note: This can be simplified - the empty string is falsey so this can just be 
```suggestion
    if (src) {
```